engines/gob/detection/detection.h

```diff
@@ -69,6 +69,10 @@ enum Features {
 	kFeaturesTrueColor = 1 << 7
 };
 
+enum AdditionalGameFlags {
+	ADGF_ENABLE_ADIBOU2_FREE_BANANAS_WORKAROUND = 1 << 0
```

Comment:
No, please name it just `GF_ENABLE_ADIBOU2_FREE_BANANAS_WORKAROUND`. AD i=stands for AdvancedDetector, and that prefix is used for telling the engine-specific flags from the global ones.